lectures/supply_demand_heterogeneity.md

```diff
@@ -68,15 +68,15 @@ $$
 (\mu_1 + \mu_2) p = \Pi^{\top}(b_1+ b_2) - \Pi^{\top} \Pi (e_1 + e_2)
 $$ (eq:old6)
 
-我们可以通过设定 $\mu_1 + \mu_2 =1$ 来规范价格，然后解方程
+我们可以使 $\mu_1 + \mu_2 =1$ 来对价格进行归一化处理，然后解方程
 
 $$
 \mu_i(p,e) = \frac{p^{\top} (\Pi^{-1} b_i - e_i)}{p^{\top} (\Pi^{\top} \Pi)^{-1} p}
 $$ (eq:old7)
 
-求得 $\mu_i, i = 1,2$。
+求得 $\mu_i, 其中 i = 1,2$。
 
-```{exercise-start}
+```{练习-start}
```

Comment:
```suggestion
```{exercise-start}
```

Reply:
admonition names will need to be in English for them to work with `Jupyter-book` build software.